test/unit/service/debtor-service.ts

```diff
@@ -335,14 +335,28 @@ describe('DebtorService', (): void => {
 
       await Fine.clear();
       await Transfer.remove(fineTransfers);
-      await FineHandoutEvent.clear();
-      await UserFineGroup.clear();
+
+      // Needed for non sqlite databases.
```

Comment:
Why is this needed? I do not understand why the FineHandoutEvent and UserFineGroup tables should be emptied after seeding.

Reply:
thats your code